extensions/functions_arithmetic.yaml

```diff
@@ -922,6 +922,69 @@ aggregate_functions:
         decomposable: MANY
         intermediate: fp64?
         return: fp64?
+  - name: "sum0"
```

Comment:
An analog could be the multiply function returning one for a set of zero elements.  Do we want to make a similar change?

Reply:
I think you’re correct. We should probably add that in another PR?

Reply:
Agreed, no need to make this PR do too much.

Reply:
👍